tests/component/test_task.py

```diff
@@ -194,3 +203,10 @@ def test___default_arguments___perform_thrmcpl_lead_offset_nulling_cal___no_erro
     ai_thermocouple_task: nidaqmx.Task,
 ) -> None:
     ai_thermocouple_task.perform_thrmcpl_lead_offset_nulling_cal()
+
+
+def test___is_task_done___task_is_done___returns_true(ai_on_demand_task: nidaqmx.Task):
```

Comment:
given = preconditions, i.e. on_demand_task or task_is_done
when = the action you are testing, ie.. is_task_done
then = the result, that part is right

Reply:
Changed to `test___on_demand_task_is_done___is_task_done___returns_true`.